addons/im_livechat/i18n/im_livechat.pot

```diff
@@ -59,6 +59,15 @@ msgstr ""
 msgid "% of Happiness"
 msgstr ""
 
+#. module: im_livechat
+#. odoo-python
+#: code:addons/im_livechat/models/discuss_channel.py:0
+#, python-format
+msgid ""
+"%(new_line)sType %(bold_start)s:shortcut%(bold_end)s to insert a canned "
```

Comment:
move to bottom to reduce the diff

Reply:
It's generated by the tool, we shouldn't mess with it.

Reply:
I saw that and I checked with the translating team and it should be fine to just replace the changed part. but since it is generated, it is also ok to just leave it as it is.

Reply:
Manually doing it is prone to mistakes, I can't see a situation where it is better than just exporting the file and never worrying about it again.

Reply:
it's just suggested only if you know where you changed the code and it reduced the diff so that it's easier to review the code. i revert the change so that the pot is non-touched.

Reply:
Agreed with Seb. Please don't change the pot files manually. The only change I'd recommend is to revert the diff on the date to avoid pointless conflicts during the forward port 🙂